test/policies/adoptable_pet_policy_test.rb

```diff
@@ -4,12 +4,28 @@
 class AdoptablePetPolicyTest < ActiveSupport::TestCase
   include PetRescue::PolicyAssertions
 
-  def setup
-    @policy = -> { AdoptablePetPolicy.new(@pet, user: @user) }
+  context "relation_scope" do
```

Comment:
This test was missing previously. The `relation_scope` was already defined previously.